release-notes/release-notes56.md

```diff
@@ -26,7 +26,10 @@ The following new features and enhancements were added for the {{site.productnam
 
 ### New `image_file_types` option for customizing recognized image file extensions
 
-The new `image_file_types` option allows different image file extensions to be recognized by the `smart_paste` image URL to embedded image functionality, as well as image file uploads.
+The new `image_file_types` option allows different image file extensions to be recognized by:
+
+- The `smart_paste` image URL to embedded image functionality
```

Comment:
```suggestion
- The `smart_paste` image URL to embedded image functionality.
```

Reply:
I just realised what this meant and it took me a good couple reads. I think if it was `Smart Paste's image URL to embedded image functionality` it would be clearer to me, but I don't think we want to do that either. Can't think of an alternative that works for now though sorry. Commenting in case anyone else can.

Reply:
Thanks @metricjs. I updated the suggestion.